packages/account-v2/src/constants/financialInformationList.ts

```diff
@@ -0,0 +1,108 @@
+export const EMPLOYMENT_VALUES = Object.freeze({
```

Comment:
can you update the PR with latest master. I see this config is already merged with @suisin-deriv PR